etc/config-template.toml

```diff
@@ -46,391 +52,474 @@
-# Interval to check whether start manual compaction for a region,
+## Interval to check whether start manual compaction for a region.
 # region-compact-check-interval = "5m"
 
-# Number of regions for each time to check.
+## Number of regions for each time to check.
 # region-compact-check-step = 100
 
-# The minimum number of delete tombstones to trigger manual compaction.
+## The minimum number of delete tombstones to trigger manual compaction.
 # region-compact-min-tombstones = 10000
 
-# The minimum percentage of delete tombstones to trigger manual compaction.
-# Should between 1 and 100. Manual compaction only triggered when the number
-# of delete tombstones exceeds region-compact-min-tombstones and the percentage
-# of delete tombstones exceeds region-compact-tombstones-percent.
+## The minimum percentage of delete tombstones to trigger manual compaction.
+## Should between 1 and 100. Manual compaction only triggered when the number
+## of delete tombstones exceeds region-compact-min-tombstones and the percentage
+## of delete tombstones exceeds region-compact-tombstones-percent.
 # region-compact-tombstones-percent = 30
 
-# Interval to check whether should start a manual compaction for lock column family,
-# if written bytes reach lock-cf-compact-threshold for lock column family, will fire
-# a manual compaction for lock column family.
+## Interval to check whether should start a manual compaction for lock column family,
+## if written bytes reach lock-cf-compact-threshold for lock column family, will fire
+## a manual compaction for lock column family.
 # lock-cf-compact-interval = "10m"
 
 # lock-cf-compact-bytes-threshold = "256MB"
 
-# Interval (s) to check region whether the data are consistent.
+## Interval (s) to check region whether the data are consistent.
 # consistency-check-interval = 0
 
-# Use delete range to drop a large number of continuous keys.
+## Whether to enable delete range feature.
+## Delete range can drop a large number of continuous keys.
 # use-delete-range = false
 
-# delay time before deleting a stale peer
+## Delay time before deleting a stale peer
 # clean-stale-peer-delay = "10m"
 
-# Interval to cleanup import sst files.
+## Interval to cleanup import SST files.
 # cleanup-import-sst-interval = "10m"
 
 [coprocessor]
-# When it is true, it will try to split a region with table prefix if
-# that region crosses tables. It is recommended to turn off this option
-# if there will be a large number of tables created.
+## When it is set to `true`, TiKV will try to split a region with table prefix if
+## that region crosses tables.
+## It is recommended to turn off this option
+## if there will be a large number of tables created.
 # split-region-on-table = true
-# When the region's size exceeds region-max-size, we will split the region
-# into two which the left region's size will be region-split-size or a little
-# bit smaller.
+## When the region's size exceeds `region-max-size`, TiKV will split the region
```

Comment:
Oops, I forget to update this related configs for batch split. Please help me to update these.😳

```toml
## For once split check, there are several split_key produced for batch.
## `batch_split_limit` limits the number of produced split-key for one batch.
# batch-split-limit = 10

## When region [a,e) size meets `region_max_size`, it will be split into
## several regions [a,b), [b,c), [c,d), [d,e). And the size of [a,b),
## [b,c), [c,d) will be `region_split_size` (maybe a little larger).
# region-max-size = "144MB"
# region-split-size = "96MB"

## When the number of keys in region [a,e) meets the `region_max_keys`,
## it will be split into two several regions [a,b), [b,c), [c,d), [d,e).
## And the number of keys in [a,b), [b,c), [c,d) will be `region_split_keys`.
# region-max-keys = 1440000
# region-split-keys = 960000
```